app/views/posts/show.html.erb

```diff
@@ -3,25 +3,21 @@
     <ul class="unorder-list">
     <li class="list-items list-content">
     <div class="post-details">
-    <p class="post-title"> Laravel</p>
+    <p class="post-title"> <%= @post.title %></p>
     <p class="quantity">
-     comments:4
-     Likes:2
+     comments:<%= @post.comments_counter %>
+     Likes:<%= @post.likes_counter %>
      </p>
     </div>
-    <p class="name-title">Hi Laravel</p>
+    <p class="name-title"><%= @post.text %></p>
     </li>
     </ul>
     <ul class="unorder-list">
     <li class="list-items list-content">
+<% @post.comments.each do |comment| %>
```

Comment:
- [ ] It will be great to use the `recent_comments` method you have created in your posts model to list the 5 most recent comments. This is a major requirement for your project to be approved